analytical_engine/java/pom.xml

```diff
@@ -122,16 +122,16 @@
             <!-- optional: limit format enforcement to just the files changed by this feature branch -->
             <!--                        <ratchetFrom>origin/main</ratchetFrom>-->
             <!-- define a language-specific format -->
-            <java>
-              <googleJavaFormat>
-                <reflowLongStrings>false</reflowLongStrings>
-                <style>AOSP</style>
-                <version>${google-java-format.version}</version>
-              </googleJavaFormat>
+<!--            <java>-->
```

Comment:
Maybe it is not a good idea to do java-format by this plugin, since the version of google-java-format used by dongze is relatively new, and requires at least Java11 in env.

So I think it is better to remove this plugin from pom, and do the format outside maven.